chess/UI/ChessGameView.swift

```diff
@@ -17,10 +17,39 @@ struct ChessGameView: View {
             .onAppear {
                 self.chessViewModel.gameAppeared()
             }
+            .errorAlert(error: self.$chessViewModel.error)
     }
 
 }
 
 #Preview {
     ChessGameView(chessViewModel: ChessGameViewModel())
 }
+
+extension View {
+    func errorAlert(error: Binding<Error?>, buttonTitle: String = "OK") -> some View {
```

Comment:
stolen from here
https://www.avanderlee.com/swiftui/error-alert-presenting/

Reply:
steal like an artist 🤌

Reply:
There is no way for me to see this rn?

Reply:
we can throw any error intentionally in `move` method for example. Just checkout on my branch and change the code a little to check

Reply:
this is how it looks like when you throw any error from model code 
![image](https://github.com/user-attachments/assets/3feedc5b-4a29-4cb6-adac-0588a3171a38)